Just.Anarchy.Test.Unit/Core/SchedulerTests.cs

```diff
@@ -106,11 +108,11 @@ await action
         public async Task ScheduleTotalDurationOverridesRepeatCount()
         {
             var expExecutionCount = 2;
-            var expTotalDuration = TimeSpan.FromSeconds(1);
+            var expTotalDuration = TimeSpan.FromSeconds(1.8);
```

Comment:
This was increased to make this test more consistent, as it's time based it was flaky on CI at times, a longer duration each interval will make it more resilient.